core/src/test/java/org/apache/james/core/MailAddressTest.java

```diff
@@ -37,47 +42,51 @@
     private static final String GOOD_ADDRESS = "server-dev@james.apache.org";
     private static final Domain GOOD_DOMAIN = Domain.of("james.apache.org");
 
-    private static final String[] GOOD_ADDRESSES = {
-            GOOD_ADDRESS,
-            GOOD_QUOTED_LOCAL_PART,
-            "server-dev@james-apache.org",
-            "server-dev@[127.0.0.1]",
-            "server-dev@#123",
-            "server-dev@#123.apache.org",
-            "server.dev@james.apache.org",
-            "\\.server-dev@james.apache.org",
-            "server-dev\\.@james.apache.org",
+    private static Stream<Arguments> goodAddresses() {
+        return Stream.of(
```

Comment:
well, `map(Arguments::of)` would be cleaner

Reply:
I see... Thanks for the feedback, will do !